container/src/main/java/com/flipkart/poseidon/filters/HystrixContextFilter.java

```diff
@@ -122,8 +122,9 @@ private void logFailedHystrixCommands(ServletRequest request) {
         HystrixRequestLog.getCurrentRequest().getAllExecutedCommands().stream().filter(
                 command -> command.isResponseTimedOut() || command.isFailedExecution() || command.isResponseShortCircuited() || command.isResponseRejected()
         ).forEach(
-                command -> logger.error("URL: {}. Global headers: {}. Command: {}. Events: {}. Exception: ",
-                        url, globalHeaders, command.getCommandKey().name(), command.getExecutionEvents(), command.getFailedExecutionException())
+                command -> logger.error("URL: {}. Global headers: {}. Command: {}. Events: {}. Exception: {}",
+                        url, globalHeaders, command.getCommandKey().name(), command.getExecutionEvents(),
+                        command.getFailedExecutionException() == null ? "" : command.getFailedExecutionException().getMessage())
```

Comment:
Avoiding logging full stack trace of exception here. We still get the error status code returned by service and the response body.
Ex: ERROR com.flipkart.poseidon.filters.HystrixContextFilter: URL: /v1/userPosts. Global headers: {X-REQUEST-ID=e0bfc75f-2f3f-4587-9      d30-dca7c2503179}. Command: sampleServiceHttpRequest. Events: [FAILURE, FALLBACK_MISSING]. Exception: **500 Session Error (URI = /users?log=true&id=1, METHOD = GET)**